src/app.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import FlashcardForm from './flashcard-form'
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      flashcards: []
+    }
+    this.saveFlashcard = this.saveFlashcard.bind(this)
+  }
+
+  saveFlashcard(event) {
```

Comment:
Try to move most of this method into your form component. `saveFlashCard` shouldn't be concerned with `event`s and element values. Instead, define a method that simply receives a new `flashCard` object and adds it to state. Push the work of building this object down into the form component.